server-x/Dockerfile

```diff
@@ -1,31 +1,37 @@
-FROM registry.access.redhat.com/ubi8-minimal AS build-env
+FROM registry.access.redhat.com/ubi8-minimal@sha256:16da4d4c5cb289433305050a06834b7328769f8a5257ad5b4a5006465a0379ff AS build-env
```

Comment:
Please remove this change as we don't have a way to handle these updates today, we can also trust these images.

Reply:
done